tools/validators/ontology_validator/yamlformat/validator/entity_type_manager.py

```diff
@@ -440,3 +440,12 @@ def _MapTypenamesBySubset(self, types_to_include, field_to_typenames,
       typenames_by_subset[field_set].update(typegroup)
 
     return typenames_by_subset
+
+
+  def GetCompleteFieldSetsOI(self) -> dict():
+    """Returns a mapping of complete field lists to EntityType strings"""
```

Comment:
Exposing an internal map is generally not preferred, but I understand how it will make it easier for you to get started, so you can leave it as-is, but add a not in the docs that exposing this is temporary until we refactor out the functionality into a reusable unit (same for other method)